ccm_web/server/src/api/api.admin.handler.ts

```diff
@@ -115,7 +136,8 @@ export class AdminApiHandler {
     const coursesWithSectionsResult = await Promise.all(coursesWithSectionsApiPromises)
     const finalResult = makeResponse<CourseWithSections>(coursesWithSectionsResult)
     if (!isAPIErrorData(finalResult)) {
-      logger.debug(`Number of sections returned: ${finalResult.length}`)
+      const sectionNum = finalResult.map(c => c.sections.length).reduce((runSum, a) => runSum + a, 0)
+      logger.debug(`Number of sections returned: ${sectionNum}`)
```

Comment:
I discovered what I added before in PR #375 is actually a bug or miscalculation. `finalResult` is a list of courses with nested section data, so if we actually want the section count, we have to do something like the new line 139.